src/components/data-entry/DataEntry.tsx

```diff
@@ -0,0 +1,196 @@
+    };
+    window.setInterval(resize, 1000);
+}
+
+function setEntryStyling(iframe: any) {
+    const iframeDocument = iframe.contentWindow.document;
+    iframeDocument.querySelector("#currentSelection").remove();
+    iframeDocument.querySelector("#header").remove();
+    iframeDocument.querySelector("html").style.overflow = "hidden";
+    iframeDocument.querySelector("#leftBar").style.display = "none";
+    iframeDocument.querySelector("#selectionBox").style.display = "none";
+    iframeDocument.querySelector("body").style.marginTop = "-55px";
+    iframeDocument.querySelector("#mainPage").style.margin = "65px 10px 10px 10px";
+    iframeDocument.querySelector("#completenessDiv").style.backgroundColor = "#5c9ccc";
+    iframeDocument.querySelector("#completenessDiv").style.border = "#5c9ccc";
+    iframeDocument.querySelector("#moduleHeader").remove();
+    autoResizeIframeByContent(iframe);
+}
+
+const waitForChildren = (el: HTMLSelectElement, datasetId: string) => {
+    return new Promise(resolve => {
+        const check = () => {
+            const option = _.filter(el.options, (option: any) => {
+                return option.value === datasetId;
+            })[0];
+            if (option) {
+                resolve();
+            } else {
+                setTimeout(check, 10);
+            }
+        };
+        check();
+    });
+};
+let dropdownItems: any[] = [];
+
+const obtainDropdownItems = (iframeDocument: HTMLIFrameElement) => {
+    const selectedPeriod = iframeDocument.querySelector("#selectedPeriodId") as HTMLSelectElement;
+    const options = selectedPeriod.options;
+    dropdownItems = [];
+    for (const option of options) {
+        const item = {
+            value: option.value,
+            text: option.text,
+        };
+        dropdownItems.push(item);
+    }
+};
+
+const setDatasetAndPeriod = async (iframe: any, datasetId: string, dropdownValue: string) => {
+    const iframeDocument = iframe.contentWindow.document;
+
+    // Constants (to be deleted)
+    const period = dropdownValue;
+
+    //get the form that we want
+    const dataSetSelector = iframeDocument.querySelector("#selectedDataSetId");
+    const periodSelector = iframeDocument.querySelector("#selectedPeriodId");
+
+    // getting datasets options and select it
+    await waitForChildren(dataSetSelector, datasetId);
+    dataSetSelector.value = datasetId;
+    dataSetSelector.onchange();
+
+    // getting periodSelector options and select it
+    await waitForChildren(periodSelector, period);
+    periodSelector.value = period;
+    periodSelector.onchange();
+
+    obtainDropdownItems(iframeDocument);
+};
+
+const getDataEntryForm = async (
+    iframe: any,
+    datasetId: string,
+    orgUnitId: any,
+    setDropdownHasValues: Function,
+    dropdownValue: string
+) => {
+    // Constants (to be deleted)
+
+    const iframeSelection = iframe.contentWindow.selection;
+
+    // await waitforElementToLoad(iframeDocument, "#selectedDataSetId");
```

Comment:
If this not of any use can we just simply delete it?

Reply:
I meant the commented out code, not the org unit selection. If you delete the selection of org unit I doubt this code actually works. Readded here -> https://github.com/EyeSeeTea/project-monitoring-app/pull/41/commits/df91ca109f96b0c4607b335a5d9a59711510cfee